### DISCUSSION TITLE
Shows dusk div when APP_ENV=testing

### DISCUSSION THREAD

**Discussion by petermaas** [wireui/wireui#456] [BUG]
I'm not sure if it is a bug so I'll put it here ;-)

When I did a release to my test environment I saw that the content of a select was shown in a div with a dusk attribute:

<img width="1481" alt="Screenshot 2022-08-29 at 21 45 56" src="https://user-images.githubusercontent.com/19660980/187472004-6b3ca437-a1ae-4c63-bdd2-c77d33a78fe8.png">

When I changed APP_ENV=testing to APP_ENV=production it was gone.

**Comment by ph7jack** [comment]
@petermaas It'll show if your app env is `testing`
The `testing` is to inform that the app is running over tests
You can use the `sandbox`, `staging`, or `production` as well. The `testing` is reserved for automated testing environment

**Reply by petermaas** [reply]
Ok, good to know, many thanks for your answer 👍🏾
